src/plugins/dashboard/public/services/dashboard_content_management/types.ts

```diff
@@ -63,6 +63,7 @@ type DashboardResolveMeta = DashboardCrudTypes['GetOut']['meta'];
 
 export interface LoadDashboardReturn {
   dashboardFound: boolean;
+  dashboardCreated?: boolean;
```

Comment:
**very small nit:** Could we maybe rename this to something like `newDashboard` or `newUnsavedDashboard`? It wasn't immediately obvious to me how we were using this based on the name, and I had to actually dig into the code to understand why this was added.

Reply:
Resolved in ac111e6c016 Renamed to `newDashboardCreated`